RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h

```diff
@@ -4,7 +4,6 @@
 // standard EDAnalyser include files
```

Comment:
Could you please modify https://github.com/cms-sw/cmssw/blob/feb29957faf4be06f28e72d2a76eef03dea1beaf/RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h#L1-L2

to

```cpp
#ifndef RecoParticleFlow_PFTracking_PFCheckHitPattern_H
#define RecoParticleFlow_PFTracking_PFCheckHitPattern_H
```